src/controllers/AuthController.ts

```diff
@@ -0,0 +1,14 @@
+import { Request, Response } from 'express'
+import { verifyToken } from '../helpers/AuthHelper'
+
+const verifyTokenController = async (req: Request, res: Response) => {
```

Comment:
You need to create a custom interface that extends the `Reqeust` interface where you need to specify the custom proprieties on this specific request for example this request body has a `token` which is a string. Too much hustle but it is worth it.

```js
export interface verifyTokenControllerRequest extends Request {
  body: { token: string }
}
```

the same can be applied to the response :thinking: ?

Also drop the `Controller` part in the function name, we already can tell what is that from the file name.

Reply:
I tried doing the same with the response interface. It doesn't work. I may have missed something tho.
Also, I still don't get why returning a custom interface won't work with the `AuthHelpers` functions. I still have to cast them to `any` which doesn't look right!